types/util.go

```diff
@@ -0,0 +1,34 @@
+package types
```

Comment:
miss license.

Reply:
Done :)

Reply:
Like the `stringSize()` method in `Java`

```java
    final static int [] sizeTable = { 9, 99, 999, 9999, 99999, 999999, 9999999,
                                      99999999, 999999999, Integer.MAX_VALUE };

    // Requires positive x
    static int stringSize(int x) {
        for (int i=0; ; i++)
            if (x <= sizeTable[i])
                return i+1;
    }
```

Maybe has 4 times faster

Reply:
@wangming1993 I don't think I get what you mean here. I think we already doing the same way here as the code you posted does, right?

Reply:
@liufuyang  Yeah, I support your code. And what I posted is in order to prove it,  java source used it and I tried it, just for approve

Reply:
@wangming1993 Yep thanks. I saw those code on stackover flow before. Do you know how to find them in the jdk source repo somewhere to actually see how exactly they did it? Thanks :)

Reply:
@liufuyang Use `IntelliJ IDEA`,  you can find `External Libraries` folder,  hope such image can help U. 

![image](https://user-images.githubusercontent.com/5611286/54603417-3874ee00-4a7f-11e9-9afe-41723107db67.png)

Reply:
@wangming1993 Aha I see. Thank you, looks nice 👍